modules/installing-ocp-agent-gather-log.adoc

```diff
@@ -28,16 +28,16 @@ $ ./openshift-install --dir <install_directory> agent wait-for bootstrap-complet
 ERROR Bootstrap failed to complete: : bootstrap process timed out: context deadline exceeded
 ----
 
-. If the output from the previous command indicates a failure, or if the bootstrap is not progressing, run the following command on node 0 and collect the output:
+. If the output from the previous command indicates a failure, or if the bootstrap is not progressing, run the following command to connect to the rendezvous host and collect the output:
 +
 [source,terminal]
 ----
-$ ssh core@<node-ip> sudo /usr/local/bin/agent-gather -O > <local_tmp_path>/agent-gather.tar.xz
+$ ssh core@<node-ip> agent-gather -O >agent-gather.tar.xz
 ----
 +
 [NOTE]
 ====
-You only need to gather data from node 0, but gathering this data from every node can be helpful.
+Most issues can be diagnosed using the data gathered from the rendezvous host, but if some hosts are not able to register, gathering this data from every host can be helpful.
```

Comment:
Just a few style guide nits for using active voice and "might" to indicate possibility:
```suggestion
You can diagnose most issues by using the data gathered from the rendezvous host, but if some hosts are not able to register, gathering this data from every host might be helpful.
```

Reply:
Hmm, yeah good point. The agent-gather is mostly for _us_ to diagnose issues though, it's unlikely that the end user will actually be able to do much of anything with it :smile:. How do we usually refer to RH support in these docs?

Reply:
That's also a good point, I forgot that this data was meant to be given to support. I _think_ this would be more appropriate way to word it in that case:
```suggestion
Red{nbsp}Hat Support can diagnose most issues using the data gathered from the rendezvous host, but if some hosts are not able to register, gathering this data from every host might be helpful.
```

If that doesn't sound right to you then you can honestly keep it the way it is, but I think changing "can" to "might" at the end of the sentence is important since our style guide tells us to use "might" to indicate possibility, and only use "can" to indicate capability (e.g. "you can use ZTP manifests for your Agent install")

Reply:
That works for me, updated. Thank you!